packages/manager/modules/iam/src/routes/onboarding.route.js

```diff
@@ -1,16 +1,28 @@
 import { onboarding as onboardingComponent } from '@iam/components';
-import { asResolve, noBreadcrumbResolve } from '@iam/resolves';
+import {
+  asResolve,
+  noBreadcrumbResolve,
+  advancedModeResolve,
+  hasPoliciesResolve,
+} from '@iam/resolves';
 
 const name = 'onboarding';
-const resolves = [noBreadcrumbResolve];
+const resolves = [noBreadcrumbResolve, advancedModeResolve];
```

Comment:
I can't find where the route needs this resolve ?
Could it be omitted ?

Reply:
It's applied